contrib/fdb-c-cmake/fdb_c.cpp

```
@@ -0,0 +1,103 @@
+#include "foundationdb/fdb_c.h"
+#include <exception>
+#include <fstream>
+#include <stdexcept>
+#include <string_view>
+#include <dlfcn.h>
+#include <unistd.h>
+#include <Common/getResource.h>
+#include <iostream>
+
+namespace
+{
+class FDBLibrary
+{
+public:
+    static FDBLibrary instance;
+
+    FDBLibrary()
+    {
+        auto data = getResource(FDB_C_RESOURCE_NAME);
+        if (data.empty())
+        {
+            std::cerr << "fdb is not loaded: libfdb_c.so is not embeded" << std::endl;
+            return;
+        }
+
+        try
+        {
+            loadDLFromMemory(data);
+            loadApis();
+        }
+        catch (std::exception & e)
+        {
+            std::cerr << "fdb is not loaded: " << e.what() << std::endl;
```

We don't allow ignoring exceptions.